### PR TITLE
feat(ecs): `interactive` option in `ContainerDefinitionOptions`

### LINKED ISSUE #24326: (cdk_ecs): interactive property missing in container definition
### Describe the bug

We are trying to use this property https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ecs.CfnTaskDefinition.ContainerDefinitionProperty.html#interactive in a fargate task but when we add it using 

https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ecs.FargateTaskDefinition.html#addwbrcontainerid-props

We get error "interactive" does not exist in type 'ContainerDefinitionOptions'. How can we solve this ?

One observation: 'interactive' property doesn't exist in non-cfn construct for container definition https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_ecs.ContainerDefinitionOptions.html

### Expected Behavior

We should be able to specify 'interactive' property

### Current Behavior

throws error

### Reproduction Steps

try to create a fargate task using cdk and add 'interactive': true to container defintion

### Possible Solution

_No response_

### Additional Information/Context

_No response_

### CDK CLI Version

2.59.0

### Framework Version

_No response_

### Node.js Version

14.x

### OS

Amazon Linux 2

### Language

Typescript

### Language Version

3.9.10

### Other information

_No response_

### PR DESCRIPTION
**Pull request by badmintoncryer**
This pull request adds the `interactive` argument to `ContainerDefinitionOptions`. This argument is used when deploying containerized applications that require the allocation of standard input (stdin) or a terminal (tty). This parameter corresponds to `OpenStdin` in the "Create a container" section of the Docker Remote API and the `--interactive` option to `docker run`.

Closes #24326.

----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/aws-cdk-lib/aws-ecs/lib/container-definition.ts [hunk 1]
```diff
@@ -251,6 +252,14 @@ export interface ContainerDefinitionOptions {
    */
   readonly hostname?: string;
 
+  /**
+   * When this parameter is true, you can deploy containerized applications that require stdin or a tty to be allocated.
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-ecs-taskdefinition-containerdefinition.html#cfn-ecs-taskdefinition-containerdefinition-interactive
+   * @default - false
+   */
+  readonly interactive?: boolean | IResolvable;
```
Comment (lpizzinidev) [inline packages/aws-cdk-lib/aws-ecs/lib/container-definition.ts:261]:
```suggestion
  readonly interactive?: boolean;
```
The property should be resolved.

File: packages/aws-cdk-lib/aws-ecs/README.md [hunk 2]
```diff
@@ -482,6 +482,19 @@ const taskDef = new ecs.TaskDefinition(this, 'TaskDef', {
 taskDef.grantRun(role);
 ```
 
+To deploy containerized applications that require the allocation of standard input (stdin) or a terminal (tty), you can set the `interactive` propertiy to `true` in `ContainerDefinitionOptions`.
```
Comment (lpizzinidev) [inline packages/aws-cdk-lib/aws-ecs/README.md:485]:
```suggestion
To deploy containerized applications that require the allocation of standard input (stdin) or a terminal (tty), use the `interactive` property.
```

File: packages/aws-cdk-lib/aws-ecs/README.md [hunk 3]
```diff
@@ -482,6 +482,19 @@ const taskDef = new ecs.TaskDefinition(this, 'TaskDef', {
...
+
+This parameter corresponds to OpenStdin in the [Create a container](https://docs.docker.com/engine/api/v1.35/#tag/Container/operation/ContainerCreate) section of the [Docker Remote API](https://docs.docker.com/engine/api/v1.35/) and the `--interactive` option to `docker run`.
```
Comment (lpizzinidev) [inline packages/aws-cdk-lib/aws-ecs/README.md:487]:
```suggestion
This parameter corresponds to `OpenStdin` in the [Create a container](https://docs.docker.com/engine/api/v1.35/#tag/Container/operation/ContainerCreate) section of the [Docker Remote API](https://docs.docker.com/engine/api/v1.35/)
and the `--interactive` option to [docker run](https://docs.docker.com/engine/reference/run/#security-configuration).
```
Let's add also the link to `docker run`.

File: packages/aws-cdk-lib/aws-ecs/lib/container-definition.ts [hunk 4]
```diff
@@ -9,6 +9,7 @@ import * as iam from '../../aws-iam';
 import * as secretsmanager from '../../aws-secretsmanager';
 import * as ssm from '../../aws-ssm';
 import * as cdk from '../../core';
+import { IResolvable } from '../../core';
```
Comment (lpizzinidev) [inline packages/aws-cdk-lib/aws-ecs/lib/container-definition.ts:12]:
```suggestion
```
Unnecessary.

**Review by lpizzinidev — CHANGES_REQUESTED** [review]
Thanks 👍
I left some comments for minor adjustments.

**Comment by badmintoncryer** [comment]
@lpizzinidev Thank you for your review! I have made the revisions as per your suggestion.

**Review by kaizencc — APPROVED** [review]

**Review by kaizencc — DISMISSED** [review]

**Comment by badmintoncryer** [comment]
@kaizencc I apologize, I didn't notice the approval and accidentally updated the branch.

**Review by kaizencc — APPROVED** [review]

**Pull request merged**